### DISCUSSION TITLE
minio configuration import/export ERROR

### DISCUSSION THREAD

**Discussion by un0of** [minio/minio#19122] [Q&A]
![check](https://github.com/minio/minio/assets/157967004/67a5a1c9-4213-4edf-a0d7-72b4aa077f1e)

![check](https://github.com/minio/minio/assets/157967004/e855b407-731d-4869-bafe-af0e521bb41a)

![check](https://github.com/minio/minio/assets/157967004/271f44ff-3f54-4ed2-83aa-5c9005724fda)

[minio-server-config-02-25-2024-09-41-02.zip](https://github.com/minio/minio/files/14395514/minio-server-config-02-25-2024-09-41-02.zip)

docker-compose
`version: '3'
services:
  minio:
    image: minio/minio:RELEASE.2024-02-24T17-11-14Z
    networks:
      net:
    ports:
      - '9001:9001'
      - '9000:9000'
    volumes:
      - 'minio_data:/data'
    environment:
      - MINIO_ROOT_USER=root
      - MINIO_ROOT_PASSWORD=7f9f4758c8a158a0a91e2307b48832d8
      - MINIO_ADDR=":9001"
      - MINIO_NOTIFY_WEBHOOK_ENABLE=on
      - MINIO_NOTIFY_WEBHOOK_ENDPOINT=http://localhost:9000
    command: server --console-address ":9001" /data
`

**Comment by klauspost** [comment]
A bit weird that for `replication_max_workers` the vlaue `500 odirect=on` is picked up.

**Reply by un0of** [reply]
please tell me how to fix this...
